app/models/miq_request_workflow.rb

```diff
@@ -792,6 +794,26 @@ def set_default_user_info
     show_fields(show_flag, [:owner_load_ldap])
   end
 
+  def set_request_values(values)
+    # Ensure that tags selected in the pre-dialog get applied to the request
+    values[:vm_tags] = (values[:vm_tags].to_miq_a + @values[:pre_dialog_vm_tags]).uniq unless @values.nil? || @values[:pre_dialog_vm_tags].blank?
+
+    if @requester.current_group
+      values[:requester_group] ||= @requester.current_group.description
+      # set owner group
+      if (email = values[:owner_email])
+        values[:owner_group] ||=
+          if email == @requester.email # optimization
+            @requester.current_group.description
+          elsif (user = User.find_by(:email => email))
+            user.current_group.description
+          end
+      end
+    end
+
+    values
```

Comment:
Copying from commit comment...

It doesn't look like you need to return values here, right?  Then you can `return unless @requester.current_group` on 801 and drop the outer "if".